Mykola_Horetskyi/3/Rock Paper Scissors.py

```diff
@@ -0,0 +1,59 @@
+import random
+
+
+def playRockPaperScissors():

```

Comment:
It's a bad idea to implement the whole game as one function. Also player wants to play the game when he executes the file. Where is your game_loop?